src/main/scala/cromwell/engine/io/gcs/GcsFileAttributes.scala

```diff
@@ -0,0 +1,25 @@
+package cromwell.engine.io.gcs
+
+import java.nio.file.Path
+import java.nio.file.attribute.{FileTime, BasicFileAttributes}
+
+class GcsFileAttributes(path: Path) extends BasicFileAttributes {
+
```

Comment:
My larger point is I see absolutely zero difference between a blank NotImplementedError and ??? since the effect is the same, however I know @kshakir disagrees. IMO if we're going to go through the effort to type out the whole exception the message should at least indicate if it's intendedto be supported in the future or simply a blank wall